ci/docker/android-emulator/non-hermetic/Dockerfile

```diff
@@ -1,7 +1,5 @@
-ARG DOCKER_REGISTRY
-FROM ${DOCKER_REGISTRY}/android/ubuntu:20.04
+FROM ubuntu:20.04
```

Comment:
I suppose it's ok for non-hermetic build, because Avito need hermetic for internal builds anyway